lib/column/datetime64.go

```diff
@@ -141,6 +141,8 @@ func (dt *DateTime64) AppendRow(v interface{}) error {
 	switch v := v.(type) {
 	case int64:
 		datetime = v
+	case *int64:
+		datetime = *v
```

Comment:
The value can be nil. It's potentially `invalid memory address or nil pointer dereference`